lib/scout/widgets/filter_menu_error.dart

```diff
@@ -1,16 +1,27 @@
+import 'package:ax_dapp/wallet/bloc/wallet_bloc.dart';
 import 'package:flutter/material.dart';
 
 class FilterMenuError extends StatelessWidget {
-  const FilterMenuError({super.key});
+  const FilterMenuError({
+    this.selectedChain = EthereumChain.sxMainnet,
```

Comment:
it would be better to use 
required this.selectedChain = EthereumChain.sxMainnet
because we have to require this param